### PR TITLE
[Ready-To-Merge]Dev mlservices 2019 11 01 - private repo pr Azure/azure-rest-api-specs-pr#902

### PR DESCRIPTION
**Pull request by vrushg-ms**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by vrushg-ms** [comment]
Already approved PR on private repo. https://github.com/Azure/azure-rest-api-specs-pr/pull/902

**Comment by vrushg-ms** [comment]
Already approved PR on private repo
https://github.com/Azure/azure-rest-api-specs-pr/pull/902

**Review by KrisBash — APPROVED** [review]
Already ARM approved in pr repo

**Comment by thoo1** [comment]
I'm part of a team that is trying to upgrade the arm-machinelearningservices package for our use.
We were using [this version before](https://github.com/Azure/azure-rest-api-specs/blob/8047a9793a7ac72fd21bec2a074987dd2e0aa218/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json)

and are trying to upgrade to [this version in the PR](https://github.com/Azure/azure-rest-api-specs/blob/8047a9793a7ac72fd21bec2a074987dd2e0aa218/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-10-01/machineLearningServices.json)

We are having issues with the operationId: "MachineLearningCompute_ListNodes"
We noticed that even in the example response for amlComputeNodes, it returns a `nextLink` but in the machineLearningServices.json file, that resource does not have the x-ms-pageable property, thus we have no way to put that `nextLink` to page through the nodes.

If someone could verify whether or not  "MachineLearningCompute_ListNodes" is pageable, that'd be great.
Note that "MachineLearningCompute_ListNodes" was pageable in the previous version (2018-11-19).

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-10-01/machineLearningServices.json [hunk 1]
```diff
@@ -0,0 +1,2861 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get compute nodes information for a compute": {
+            "$ref": "./examples/amlComputeListNodes.json"
```
Comment (thoo1) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-10-01/machineLearningServices.json]: for reference to my comment above: this is what I'm talking about; the example shows that the response has a `nextLink` but this operation does not have property "x-ms-pageable", thus it seems to say the operation is not pageable, could someone confirm whether it is or isn't

Comment (vrushg-ms) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-10-01/machineLearningServices.json]: Thank you for bringing this to our attention. I’ll take a look and update the thread.

Comment (thoo1) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-10-01/machineLearningServices.json]: thanks, if you need to contact me, my alias is my username without the 1

Comment (vrushg-ms) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-10-01/machineLearningServices.json]: Let's talk offline then.

**Comment by MyronFanQiu** [comment]
@vrushg-ms Hello. Once the pr is ready, please ping me and help you merge this PR.

**Comment by MyronFanQiu** [comment]
@vrushg-ms Hello. Is this PR ready?

**Comment by MyronFanQiu** [comment]
@vrushg-ms Hello. Is this PR ready?

**Comment by vrushg-ms** [comment]
> @vrushg-ms Hello. Is this PR ready?

We want to merge this on  November 1st.

**Review by mmyyrroonn — APPROVED** [review]

**Comment by vrushg-ms** [comment]
@MyronFanQiu , The PR is now ready for merging

**Pull request merged**